### DISCUSSION TITLE
stat-computing-r-python/part-gen-prog/01-basic-var-types

### DISCUSSION THREAD

**Discussion by giscus** [srvanderplas/stat-computing-r-python#18] [General]
# stat-computing-r-python/part-gen-prog/01-basic-var-types

https://srvanderplas.github.io/stat-computing-r-python/part-gen-prog/01-basic-var-types.html

**Comment by emitanaka** [comment]
This is very nice Susan! Have you thought of a license for this? Perhaps @robjhyndman may like to use as a reference for Advanced R Programmimg unit at Monash.

**Comment by srvanderplas** [comment]
Anyone is welcome to use :) It's got a GPL3 license file. I'd also welcome pull requests!
